Include/internal/pycore_opcode_metadata.h

```diff
@@ -2109,8 +2117,9 @@ const struct opcode_metadata _PyOpcode_opcode_metadata[266] = {
     [LOAD_ATTR_WITH_HINT] = { true, INSTR_FMT_IBC00000000, HAS_ARG_FLAG | HAS_NAME_FLAG | HAS_DEOPT_FLAG | HAS_EXIT_FLAG },
     [LOAD_BUILD_CLASS] = { true, INSTR_FMT_IX, HAS_ERROR_FLAG | HAS_ESCAPES_FLAG },
     [LOAD_COMMON_CONSTANT] = { true, INSTR_FMT_IB, HAS_ARG_FLAG },
-    [LOAD_CONST] = { true, INSTR_FMT_IB, HAS_ARG_FLAG | HAS_CONST_FLAG | HAS_PURE_FLAG },
```

Comment:
Sort of unrelated... do we use `pure` at all? Or maybe we plan to, in the mythical partial evaluator?

Reply:
Nope. I asked Mark to remove it some time back. However, he didn't want churn (adding something, then removing it).

I can't see the partial evaluator using it though. `HAS_ESCAPES_FLAG` is a strong enough guarantee. We should just remove it to save the bit.